src/apps/content-editor/src/app/components/Nav/ItemsFilter.js

```diff
@@ -1,23 +1,29 @@
-import { Search } from "@zesty-io/core/Search";
-import cx from "classnames";
-import { useSelector } from "react-redux";
-import styles from "./ContentNav.less";
+import TextField from "@mui/material/TextField";
+import InputAdornment from "@mui/material/InputAdornment";
+import SearchIcon from "@mui/icons-material/Search";
 
 const ItemsFilter = (props) => {
-  const ui = useSelector((state) => state.ui);
   return (
-    <Search
-      className={cx(
-        styles.SearchModels,
-        ui.contentNavHover ? "" : styles.HideIcon
-      )}
+    <TextField
       name="itemsFilter"
       placeholder="Filter items by name, zuid or path"
-      value={props.searchTerm}
```

Comment:
I believe you removed this by accident

Reply:
When I add the `value={props.searchTerm}` back in I get a search bug I can't delete the term all the way? 

<img width="477" alt="Screen Shot 2022-06-22 at 7 01 51 AM" src="https://user-images.githubusercontent.com/22800749/175048461-c1ab63a0-53ab-43fe-bd76-fc25f6edb40e.png">